docs/guide/protocols/plainmd.rst

```diff
@@ -1,4 +1,55 @@
 Plain MD Protocol
 =================
 
-issue #743
\ No newline at end of file
+Overview
+--------
+
+The :class:`.PlainMDProtocol` enables the user to run an MD simulation of a ``ChemicalSystem``, which can contain e.g. a solvated protein-ligand complex, or a ligand and water.
+
+TODO: Later add ref to ChemicalSystem section
+
+Scientific Details
+------------------
+
+The :class:`.PlainMDProtocol` runs MD simulations of a system either in solvent or vacuum, depending on the input provided by the user in the `ChemicalSystem`.
+The protocol applies a 
+`LangevinMiddleIntegrator <http://docs.openmm.org/development/api-python/generated/openmm.openmm.LangevinMiddleIntegrator.html>`_ 
+which uses Langevin dynamics, with the LFMiddle discretization [1]_.  
+
+Simulation Steps
+~~~~~~~~~~~~~~~~
+
+If there is a ``SolventComponent`` in the ``ChemicalSystem``, the each Protocol simulation Unit carries out the following steps:
+
+1. Parameterize the system using `OpenMMForceFields <https://github.com/openmm/openmmforcefields>`_ and `Open Force Field <https://github.com/openforcefield/openff-forcefields>`_.
+2. Minimize the system
+3. Equilibrate in the canonical ensemble
+4. Equilibrate and production simulate the system (under NPT conditions using a MonteCarloBarostat to maintain constant pressure)
```

Comment:
```suggestion
3. Equilibrate in the canonical (NVT) ensemble
4. Equilibrate and production simulate the system under isobaric-isothermal (NPT) conditions using a MonteCarloBarostat to maintain constant pressure
```
😅 we don't necessary need to go this far, just if we wanted to spell out the ensembles we can do this